worlds/tunic/docs/en_Tunic.md

```diff
@@ -1,15 +1,20 @@
 # TUNIC
 
-## Where is the settings page?
+## Where is the options page?
 
-The [player settings page for this game](../player-settings) contains all the options you need to configure and export a config file.
+The [player options page for this game](../player-options) contains all the options you need to configure and export a config file.
+
+## I haven't played TUNIC before.
+
+**Play vanilla first.** It is **_heavily discouraged_** to play this randomizer before playing the vanilla game, moreso than most other games in Archipelago.
```

Comment:
```suggestion
**Play vanilla first.** It is **_heavily discouraged_** to play this randomizer before playing the vanilla game.
```
I think this is bad. You probably haven't played them all and you can't possibly know which games will come next.